tf_files/cloud.tf

```diff
@@ -1,3 +1,11 @@
+terraform {
+    backend "s3" {
+        bucket = "cdis-terraform-state"
+        key    = "cdis-vpc/terraform.tfstate"
```

Comment:
the backend configuration cannot contain interpolations. So, we need to input the whole key or hardcode it. Like I did in `terraform.tfvars` for `access_key` and `secret_key`